lib/promote_release.js

```diff
@@ -0,0 +1,334 @@
+    const keyPath = await cli.prompt(
+      `Please enter the path to your ssh key (Default ${defaultKeyPath}): `,
+      { questionType: 'input', defaultAnswer: defaultKeyPath });
+    await this.promoteAndSignRelease(keyPath);
+
+    cli.separator();
+    cli.ok(`Release promotion for ${version} complete.\n`);
+    cli.info(
+      'To finish this release, you\'ll need to: \n' +
+      ` 1) Check the release at: https://nodejs.org/dist/v${version}\n` +
+      ' 2) Create the blog post for nodejs.org\n' +
+      ' 3) Create the release on GitHub\n' +
+      'Finally, proceed to Twitter and announce the new release!');
+  }
+
+  async verifyPRAttributes() {
+    const { cli, prid, owner, repo } = this;
+
+    const credentials = await auth({ github: true });
+    const request = new Request(credentials);
+
+    const data = new PRData({ prid, owner, repo }, cli, request);
+    await data.getAll();
+
+    const checker = new PRChecker(cli, data, { prid, owner, repo });
+    const jenkinsReady = checker.checkJenkinsCI();
+    const githubCIReady = checker.checkGitHubCI();
+    const isApproved = checker.checkReviewsAndWait(new Date(), false);
+
+    return {
+      jenkinsReady,
+      githubCIReady,
+      isApproved
+    };
+  }
+
+  async parseDataFromReleaseCommit() {
+    const { cli } = this;
+
+    const releaseCommitMessage = runSync(
+      'git', ['log', '-n', '1', '--pretty=format:\'%s\'']).trim();
+
+    const components = releaseCommitMessage.split(' ');
+
+    // Parse out release date.
+    const match = components[0].match(/\d{4}-\d{2}-\d{2}/);
+    if (!match) {
+      cli.error(`Release commit contains invalid date: ${components[0]}`);
+      return;
+    }
+    this.date = match[0];
+
+    // Parse out release version.
+    const version = semver.clean(components[2]);
+    if (!semver.valid(version)) {
+      cli.error(`Release commit contains invalid semantic version: ${version}`);
+      return;
+    }
+
+    this.version = version;
+    this.stagingBranch = `v${semver.major(version)}.x-staging`;
+    this.versionComponents = {
+      major: semver.major(version),
+      minor: semver.minor(version),
+      patch: semver.patch(version)
+    };
+
+    // Parse out LTS status and codename.
+    if (components.length === 5) {
+      this.isLTS = true;
+      this.ltsCodename = components[3];
+    }
+  }
+
+  getCommitSha(position = 0) {
+    return runSync('git', ['rev-parse', `HEAD~${position}`]).trim();
+  }
+
+  get owner() {
+    return this.config.owner || 'nodejs';
+  }
+
+  get repo() {
+    return this.config.repo || 'node';
+  }
+
+  get username() {
+    return this.config.username;
+  }
+
+  async secureTagRelease() {
+    const { version, isLTS, ltsCodename } = this;
+
+    const secureTag = path.join(
+      __dirname,
+      '../node_modules/.bin/git-secure-tag' + (isWindows ? '.cmd' : '')
+    );
+
+    const releaseInfo = isLTS ? `'${ltsCodename}' (LTS)` : '(Current)';
```

Comment:
```suggestion
    const releaseInfo = isLTS ? `${ltsCodename} (LTS)` : '(Current)';
```

There are already quotes around the codename in the commit message